packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -267,16 +276,26 @@ class ValidatorsList extends React.PureComponent<Props, State> {
       .sort((a, b) => b.id - a.id)
       .sort((a, b) => compare(dAccessor(a), dAccessor(b)))
       .sort((a, b) => dir * compare(accessor(a), accessor(b)))
+      .sort((a, b) => this.isPinned(b) - this.isPinned(a))
+  }
+
+  isPinned({ address }: any) {
```

Comment:
nit: type as `{addres: string}`

Reply:
Not possible at all, because it is used in the sorting that doesn't have enough typing information (and I didn't want to over typed it, in the end, the data is an array of key/values).

Reply:
ah ok